lib/std/os/linux.zig

```diff
@@ -906,7 +907,7 @@ pub fn mmap(address: ?[*]u8, length: usize, prot: usize, flags: MAP, fd: i32, of
             prot,
             @as(u32, @bitCast(flags)),
             @bitCast(@as(isize, fd)),
-            @truncate(@as(u64, @bitCast(offset)) / MMAP2_UNIT),
```

Comment:
Don't forget to remove the definitions of `MMAP2_UNIT` in the other arch bits files.